openapi/nexus.json

```diff
@@ -18116,6 +18116,25 @@
           "sum_of_samples"
         ]
       },
+      "Duration": {
+        "type": "object",
+        "properties": {
+          "nanos": {
+            "type": "integer",
+            "format": "uint32",
+            "minimum": 0
+          },
+          "secs": {
+            "type": "integer",
+            "format": "uint64",
+            "minimum": 0
+          }
+        },
+        "required": [
+          "nanos",
+          "secs"
+        ]
+      },
```

Comment:
You can see by this addition that this is the first time we're sticking `std::time::Duration` directly in an API response. I imagine we have other places in the API where we represent durations, but they don't use this format. If so, it might be better to do whatever they currently do. Another point in favor of trivial `views.rs` versions of the query summary structs in order to control the generated API schemas without messing up what the oxql crate uses internally.